ur_description/launch/view_ur10e.launch

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0"?>
 <launch>
-  <include file="$(find ur_e_description)/launch/ur10e_upload.launch"/>
+  <include file="$(find ur_description)/launch/ur10e_upload.launch"/>
 
   <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" >
     <param name="use_gui" value="true"/>
   </node>
   <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
```

Comment:
Not quite part of this PR, but this should probably be replaced to use `robot_state_publisher` instead of the deprecated `state_publisher`.